apps/src/lib/node/ledger/shell/vote_extensions/val_set_update.rs

```diff
@@ -50,56 +55,32 @@ where
     pub fn validate_valset_upd_vext_and_get_it_back(
         &self,
         ext: validator_set_update::SignedVext,
-        last_height: BlockHeight,
+        signing_epoch: Epoch,
     ) -> std::result::Result<
         (token::Amount, validator_set_update::SignedVext),
         VoteExtensionError,
     > {
-        #[cfg(feature = "abcipp")]
-        if ext.data.block_height != last_height {
+        if self.storage.last_height.0 == 0 {
             tracing::error!(
-                ext_height = ?ext.data.block_height,
-                ?last_height,
-                "Validator set update vote extension issued for a block \
-                 height different from the expected last height.",
+                "Dropping validator set update vote extension issued at \
+                 genesis"
```

Comment:
```suggestion
                "Cannot validate validator set update vote extension as have \
                 not yet synced any blocks"
```
or some sort of similar message that indicates we can't validate as we haven't synced any blocks yet (rather than it being anything to do with the valset upd vext itself)